zerver/views/streams.py

```diff
@@ -175,11 +167,13 @@ def json_make_stream_private(request, user_profile, stream_name=REQ()):
 @require_realm_admin
 @has_request_variables
 def update_stream_backend(request, user_profile, stream_name,
-                          description=REQ(validator=check_string, default=None)):
+                          description=REQ(validator=check_string, default=None),
+                          new_name=REQ(default=None)):
```

Comment:
You should add `validator=check_string` here here so that it will verify any `new_name` argument is valid.